lib/dolla/configuration.rb

```diff
@@ -0,0 +1,22 @@
+module Dolla
+  class Configuration
+    attr_accessor :business_unit, :payment_gateway_code, :cancel_gateway_code, :currency,
+    :client_id, :admin_email, :transaction_type, :platform, :payment, :cancellation,
+    :rijndael_key, :hmac_key
+
+    def initialize
+      @business_unit = nil
+      @payment_gateway_code = nil
+      @cancel_gateway_code = nil
+      @currency = nil
+      @client_id = nil
+      @admin_email = nil
+      @transaction_type = nil
+      @platform = nil
+      @payment = nil
+      @cancellation = nil
+      @rijndael_key = nil
+      @hmac_key = nil
```

Comment:
Why do you initialize the ivars here?, they will be `nil` by default

Reply:
I set these values to `nil` for every new instance of the `Configuration` class, this indicates that these keys are required and are not provided by me, so they must be provided by the user.

Reply:
how about raising a `Dolla::MIssingArgument` exception or something similar if they are missing?